ml4gw/waveforms/ringdown.py

```diff
@@ -0,0 +1,90 @@
+import torch
+from torch import Tensor
+from ml4gw.types import ScalarTensor
+
+class Ringdown(torch.nn.Module):
+    """
+    Callable class for generating ringdown waveforms.
+
+    Args:
+        sample_rate: Sample rate of waveform
+        duration: Duration of waveform
+    """
+
+    def __init__(self, sample_rate: float, duration: float):
+        super().__init__()
+        # determine times based on requested duration and sample rate
+        # and shift so that the waveform is centered at t=0
+
+        num = int(duration * sample_rate)
+        times = torch.arange(num, dtype=torch.float64) / sample_rate
+        times -= duration / 2.0
+
+        self.register_buffer("times", times)
+
+    def __call__(
+        self,
+        frequency: ScalarTensor,
+        quality: ScalarTensor,
+        amplitude: ScalarTensor,
+        phase: ScalarTensor,
+        inclination: ScalarTensor,
+        #distance: ScalarTensor,  
```

Comment:
Not sure why equations 7.1 and 7.2 don't have the distance in them, but they definitely should both be divided by `r`. Maybe that gets taken into account somewhere else in the original program, but we'll want to have it here.